src/api-service/__app__/onefuzzlib/notifications/github_issues.py

```diff
@@ -107,10 +112,18 @@ def github_issue(
     config: GithubIssueTemplate,
     container: Container,
     filename: str,
-    report: Optional[Report],
+    report: Optional[Union[Report, RegressionReport]],
 ) -> None:
     if report is None:
         return
+    if isinstance(report, RegressionReport):
+        logging.info(
```

Comment:
If we expect this to come from user error, how about:
```suggestion
        logging.warning(
```

Reply:
In the near future (once this is merged) I intend to work on the ADO/Github integration to make these features, rather than "not supported yet" alerts.